frontend/src/components/Shared/HeroBillboard/HeroBillboard.jsx

```
@@ -1,103 +1,154 @@
-          alt={content.title} 
-          className="w-full h-full object-cover opacity-70"
+          src={getThumbnailUrl()} 
+          alt={video.title} 
+          className="w-full h-full object-cover"
           onError={(e) => {
-            e.target.src = `/api/placeholder/1200/600?text=${encodeURIComponent(content.title)}`;
+            e.target.onerror = null;
+            e.target.src = `/api/placeholder/1200/600?text=${encodeURIComponent(video.title || 'Featured Video')}`;
           }}
         />
+        <div className="absolute inset-0 bg-gradient-to-r from-gray-900 via-transparent to-gray-900 opacity-60"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)